src/PublicDashboard.tsx

```diff
@@ -57,7 +57,7 @@ export class PublicDashboard {
   private readonly accessKey: string;
 
   /** Dashboards API url */
-  private dashboardsApiUrl = 'blob-service.us-west-2.prod.aws.keen.io';
+  private dashboardsApiUrl = 'dashboard-service.us-west-2.test.aws.keen.io';
```

Comment:
We should use here `dashboard-service.k-n.io` by default

Reply:
for both `DashboardCreator` and `PublicDashboard`?